CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   * Make `PopupBridgeClient#POPUP_BRIDGE_NAME` package-private
   * Make `PopupBridgeClient#POPUP_BRIDGE_URL_HOST` package-private
   * Remove `PopupBridgeLifecycleObserver`
+  * Replace `FragmentActivity` with `ComponentActivity` in `PopupBridgeClient`'s constructor
```

Comment:
Super basic question here but what is the advantage to replacing the `FragmentActivity` with `ComponentActivity`?

Reply:
`FragmentActivity` extends `ComponentActivity`. So this change makes the merchant integration less prescriptive!